tests/testfiles/CommonMark_0.30/html_blocks/021.test

```diff
@@ -9,5 +9,7 @@
 <del>*foo*</del>
 >>>
 documentBegin
-inlineHtmlTag: <del>(emphasis: foo)</del>
+inlineHtmlTag: <del>
+emphasis: foo
+inlineHtmlTag: </del>
```

Comment:
In inline HTML content, we only recognize tags rather than entire elements.
See <https://witiko.github.io/markdown/#html-tag-and-element-renderers>.